--- ircd-hybrid-7.0/include/config.h	2003-12-14 02:18:47.000000000 +0100
+++ ircd-hybrid-7.0/include/config.h	2003-12-14 02:29:01.000000000 +0100
@@ -27,6 +27,8 @@
 
 #include "setup.h"
 
+#define IRC_UID	75
+#define IRC_GID	75
 
 /*
  * IRCD-HYBRID-7 COMPILE TIME CONFIGURATION OPTIONS
@@ -92,6 +94,7 @@
 #define BINPATH         "IRCD$BINDIR:"
 #define ETCPATH         "IRCD$CONFDIR:"
 #define LOGPATH         "IRCD$LOGDIR:"
+#define RUNPATH		"IRCD$RUNDIR:"
 
 #undef  MODPATH
 #undef  AUTOMODPATH
@@ -106,7 +109,7 @@
 #define RPATH   ETCPATH "IRCD.RSA"              /* RSA private key file */
 #define MPATH   ETCPATH "IRCD.MOTD"             /* MOTD filename */
 #define LPATH   LOGPATH "IRCD.LOG"              /* logfile */
-#define PPATH   ETCPATH "IRCD.PID"              /* pid file */
+#define PPATH   RUNPATH "IRCD.PID"              /* pid file */
 #define HPATH   ETCPATH "OPERS.TXT"             /* oper help file */
 #define UHPATH  ETCPATH "USERS.TXT"             /* user help file */
 #define OPATH   ETCPATH "OPERS.MOTD"            /* oper MOTD file */
@@ -132,15 +135,16 @@
  */
 
 /* dirs */
-#define DPATH   IRCD_PREFIX                                                     
-#define BINPATH IRCD_PREFIX "/bin/"
-#define MODPATH IRCD_PREFIX "/modules/"
-#define AUTOMODPATH IRCD_PREFIX "/modules/autoload/"
-#define MSGPATH IRCD_PREFIX "/messages/"
-#define ETCPATH IRCD_PREFIX "/etc"
-#define LOGPATH IRCD_PREFIX "/logs"
-#define UHPATH   IRCD_PREFIX "/help/users"
-#define HPATH  IRCD_PREFIX "/help/opers"
+#define DPATH		"/etc/ircd/"                                                  
+#define BINPATH		"/usr/sbin"
+#define MODPATH		"/usr/lib/ircd/modules"
+#define AUTOMODPATH	MODPATH "/autoload"
+#define MSGPATH		"/usr/share/locale"
+#define ETCPATH		"/etc/ircd"
+#define LOGPATH		"/var/log/ircd"
+#define HPATH		"/usr/lib/ircd/help/opers"
+#define UHPATH	        "/usr/lib/ircd/help/users"
+#define RUNPATH		"/var/run/ircd"
 
 /* files */
 #define SPATH   BINPATH "/ircd"                 /* ircd executable */
@@ -152,7 +156,7 @@
 #define RPATH   ETCPATH "/ircd.rsa"             /* ircd rsa private keyfile */
 #define MPATH   ETCPATH "/ircd.motd"            /* MOTD file */
 #define LPATH   LOGPATH "/ircd.log"             /* ircd logfile */
-#define PPATH   ETCPATH "/ircd.pid"             /* pid file */
+#define PPATH   RUNPATH "/ircd.pid"             /* pid file */
 #define OPATH   ETCPATH "/opers.motd"           /* oper MOTD file */
 #define LIPATH  ETCPATH "/links.txt"            /* cached links file */
 #endif /* !VMS */
@@ -160,7 +164,7 @@
 /* WANT_GETTEXT - toggle gettext support.
  * NOTE: if configure doesn't detect gettext, this won't do anything.
  */
-#define WANT_GETTEXT
+#undef WANT_GETTEXT
  
 /* Ignore bogus timestamps from other servers. Yes this will desync
  * the network, but it will allow chanops to resync with a valid non TS 0
