--- ircd-hybrid-6.3.1/src/Makefile.in~	Fri May  3 17:53:36 2002
+++ ircd-hybrid-6.3.1/src/Makefile.in	Fri May  3 17:53:36 2002
@@ -12,8 +12,7 @@
 CPPFLAGS = ${INCLUDES} @CPPFLAGS@
  
 # For developers
-# CFLAGS= -g -O2 -Wall
-#
+CFLAGS= @CFLAGS@
 
 SRCS = \
 	adns.c \
--- ircd-hybrid-6.3.1/Makefile.in~	Fri May  3 17:53:06 2002
+++ ircd-hybrid-6.3.1/Makefile.in	Fri May  3 17:53:06 2002
@@ -22,7 +22,7 @@
 RM=@RM@
 
 # Default CFLAGS
-CFLAGS = -g -O2 -DNDEBUG
+CFLAGS = @CFLAGS@
 
 # Default make flags - you may want to uncomment this on a multicpu machine
 #MFLAGS = -j 4
