--- ircd-hybrid-7beta15.orig/include/config.h	Wed Apr  3 15:10:29 2002
+++ ircd-hybrid-7beta15/include/config.h	Sun May 26 11:24:28 2002
@@ -27,6 +27,8 @@
 
 #include "setup.h"
 
+#define IRC_UID	75
+#define IRC_GID	75
 
 /*
  * IRCD-HYBRID-7 COMPILE TIME CONFIGURATION OPTIONS
@@ -132,15 +134,15 @@
  */
 
 /* dirs */
-#define DPATH   IRCD_PREFIX                                                     
-#define BINPATH IRCD_PREFIX "/bin/"
-#define MODPATH IRCD_PREFIX "/modules/"
-#define AUTOMODPATH IRCD_PREFIX "/modules/autoload/"
-#define MSGPATH IRCD_PREFIX "/messages/"
-#define ETCPATH IRCD_PREFIX "/etc"
-#define LOGPATH IRCD_PREFIX "/logs"
-#define UHPATH   IRCD_PREFIX "/help/users"
-#define HPATH  IRCD_PREFIX "/help/opers"
+#define DPATH		"/etc/ircd/"                                                  
+#define BINPATH		"/usr/sbin"
+#define MODPATH		"/usr/lib/ircd/modules"
+#define AUTOMODPATH	MODPATH "/autoload"
+#define MSGPATH		"/usr/share/locale"
+#define ETCPATH		"/etc/ircd"
+#define LOGPATH		"/var/log/ircd"
+#define HPATH		"/usr/lib/ircd/help"
+#define UHPATH		HPATH
 
 /* files */
 #define SPATH   BINPATH "/ircd"                 /* ircd executable */
@@ -160,7 +162,7 @@
 /* WANT_GETTEXT - toggle gettext support.
  * NOTE: if configure doesn't detect gettext, this won't do anything.
  */
-#define WANT_GETTEXT
+#undef WANT_GETTEXT
  
 /* Ignore bogus timestamps from other servers. Yes this will desync
  * the network, but it will allow chanops to resync with a valid non TS 0
