--- ircd-hybrid-7beta15/src/linebuf.c.orig	Mon Apr 15 02:05:45 2002
+++ ircd-hybrid-7beta15/src/linebuf.c	Thu Sep  5 10:37:24 2002
@@ -574,7 +574,7 @@
     va_end(prefix_args);
   }
 
-  if (va_args)
+  if (format)
   {
     len += vsnprintf((bufline->buf + len), (BUF_DATA_SIZE - len), format,
                     va_args);
--- ircd-hybrid-7beta15/src/send.c.orig	Wed Mar 13 05:21:42 2002
+++ ircd-hybrid-7beta15/src/send.c	Thu Sep  5 10:37:25 2002
@@ -1103,6 +1103,7 @@
   char nbuf[IRCD_BUFSIZE*2];
   dlink_node *ptr;
   va_list args;
+  va_list dummy;
   buf_head_t linebuf;
 
   va_start(args, pattern);
@@ -1123,7 +1124,7 @@
     if(client_p->umodes & flags)
     {
       linebuf_newbuf(&linebuf);
-      linebuf_putmsg(&linebuf, NULL, NULL,
+      linebuf_putmsg(&linebuf, NULL, dummy,
                      ":%s NOTICE %s :*** Notice -- %s", me.name,
                      client_p->name, nbuf);
 
