--- ircd-hybrid-7beta18/src/send.c.orig	Wed Mar  5 21:44:59 2003
+++ ircd-hybrid-7beta18/src/send.c	Wed Mar  5 21:44:41 2003
@@ -1052,6 +1052,7 @@
   dlink_node *ptr;
   dlink_node *ptr_next;
   va_list args;
+  va_list dummy;
   buf_head_t linebuf;
 
   va_start(args, pattern);
@@ -1073,7 +1074,7 @@
     if(client_p->umodes & flags)
     {
       linebuf_newbuf(&linebuf);
-      linebuf_putmsg(&linebuf, NULL, NULL,
+      linebuf_putmsg(&linebuf, NULL, dummy,
                      ":%s NOTICE %s :*** Notice -- %s", me.name,
                      client_p->name, nbuf);
 
