--- ircd-hybrid-7beta15/autoconf/configure.in.orig	Sun Apr 28 15:03:44 2002
+++ ircd-hybrid-7beta15/autoconf/configure.in	Thu Sep  5 10:50:32 2002
@@ -133,7 +133,7 @@
 dnl NOTE: jv says it must be added to the *END*, because things like
 dnl "gcc -O9 -O2" will result in -O2 getting preference.  How stupid.
 if test "x$CFLAGS" != "x"; then
-  IRC_CFLAGS="$IRC_CFLAGS $CPPFLAGS"
+  IRC_CFLAGS="$IRC_CFLAGS $CFLAGS"
 fi
 
 AC_ISC_POSIX
