--- ircd-hybrid-6.3.1/autoconf/configure.in~	Fri May  3 17:50:04 2002
+++ ircd-hybrid-6.3.1/autoconf/configure.in	Fri May  3 17:50:04 2002
@@ -22,9 +22,9 @@
 
 dnl Checks for libraries.
 AC_CHECK_LIB(socket, socket)
-AC_CHECK_FUNC(crypt,, AC_CHECK_LIB(descrypt, crypt,,AC_CHECK_LIB(crypt, crypt,,)))
-AC_CHECK_FUNC(inet_ntoa,, AC_CHECK_LIB(nsl, inet_ntoa,,))
-AC_CHECK_FUNC(inet_aton,, AC_CHECK_LIB(resolv, inet_aton,,))
+AC_CHECK_FUNC(crypt,[], AC_CHECK_LIB(descrypt, crypt,[],AC_CHECK_LIB(crypt, crypt,[],)))
+AC_CHECK_FUNC(inet_ntoa,[], AC_CHECK_LIB(nsl, inet_ntoa,[],))
+AC_CHECK_FUNC(inet_aton,[], AC_CHECK_LIB(resolv, inet_aton,[],))
 
 dnl Checks for header files.
 AC_HEADER_STDC
@@ -75,7 +75,7 @@
 #endif
 	exit(1);
 }
-], AC_DEFINE(NBLOCK_POSIX) AC_MSG_RESULT( O_NONBLOCK ),, AC_MSG_RESULT( cross compiling ))
+], AC_DEFINE(NBLOCK_POSIX) AC_MSG_RESULT( O_NONBLOCK ),[], AC_MSG_RESULT( cross compiling ))
 
 dnl try mmap() support
 AC_FUNC_MMAP
@@ -109,7 +109,7 @@
 #endif
 	exit(1);
 }
-], AC_DEFINE(NBLOCK_BSD) AC_MSG_RESULT( O_NDELAY ),, AC_MSG_RESULT(unable to test))
+], AC_DEFINE(NBLOCK_BSD) AC_MSG_RESULT( O_NDELAY ),[], AC_MSG_RESULT(unable to test))
 
 AC_TRY_RUN(
 [
@@ -140,7 +140,7 @@
 #endif /* !NBLOCK_POSIX && !NBLOCK_BSD && FIONBIO */
 	exit(1);
 }
-], AC_DEFINE(NBLOCK_SYSV) AC_MSG_RESULT(FIONBIO),, AC_MSG_RESULT(unable to test))
+], AC_DEFINE(NBLOCK_SYSV) AC_MSG_RESULT(FIONBIO),[], AC_MSG_RESULT(unable to test))
 
 dnl taken from ircd's Config script
 AC_MSG_CHECKING(signal implementation)
