--- ircd-hybrid-6.3.1.orig/include/config.h	Sun Feb 17 06:58:09 2002
+++ ircd-hybrid-6.3.1/include/config.h	Fri May  3 20:42:30 2002
@@ -39,6 +39,8 @@
  *
  */
 
+#define IRC_UID 75
+#define IRC_GID 75
 /***************** MAKE SURE THIS IS CORRECT!!!!!!!!! **************/
 /* ONLY EDIT "HARD_FDLIMIT_" and "INIT_MAXCLIENTS" */
 
@@ -68,7 +70,7 @@
  *       will be preallocated for the entire whowas array when ircd is started.
  *       You will want to crank this down if you are on a small net.
  */
-#define NICKNAMEHISTORYLENGTH 15000
+#define NICKNAMEHISTORYLENGTH 1000
 
 /* Don't change this... */
 #define HARD_FDLIMIT    (HARD_FDLIMIT_ - 10)
@@ -100,16 +102,16 @@
  *
  */
 
-#define DPATH   "/usr/local/ircd/"
-#define SPATH   "/usr/local/ircd/ircd"
-#define CPATH   "ircd.conf"
-#define KPATH   "kline.conf"
-#define DLPATH  "kline.conf"
-#define MPATH   "ircd.motd"
-#define LPATH   "ircd.log"
-#define PPATH   "ircd.pid"
-#define HPATH   "opers.txt"
-#define OPATH   "opers.motd"
+#define DPATH   "/etc/ircd/" /* sad but true */
+#define SPATH   "/usr/sbin/ircd"
+#define CPATH   "/etc/ircd/ircd.conf"
+#define KPATH   "/etc/ircd/kline.conf"
+#define DLPATH  "/etc/ircd/kline.conf"
+#define MPATH   "/etc/ircd/ircd.motd"
+#define LPATH   "/var/log/ircd/ircd.log"
+#define PPATH   "/var/lib/ircd/ircd.pid"
+#define HPATH   "/etc/ircd/opers.txt"
+#define OPATH   "/etc/ircd/opers.motd"
 
 /* HIDE_OPS
  * Define this to prevent non chanops from seeing what ops a channel has
@@ -208,8 +210,8 @@
  *
  * These need to be defined if you want to use SYSLOG logging, too.
  */
-#define FNAME_USERLOG "/usr/local/ircd/users"
-#define FNAME_OPERLOG "/usr/local/ircd/opers"
+#define FNAME_USERLOG "/var/log/ircd/users"
+#define FNAME_OPERLOG "/var/log/ircd/opers"
 
 /* RFC1035_ANAL
  * Defining this causes ircd to reject hostnames with non-compliant chars.
@@ -533,7 +535,7 @@
 /* MAXSENDQLENGTH - Max amount of internal send buffering
  * Max amount of internal send buffering when socket is stuck (bytes)
  */
-#define MAXSENDQLENGTH 9000000    /* Recommended value: 9000000 for EFnet */
+#define MAXSENDQLENGTH 500000	/* Recommended value: 9000000 for EFnet */
 
 /*  BUFFERPOOL - the maximum size of the total of all sendq's.
  *  Recommended value is 4 times MAXSENDQLENGTH.
@@ -756,7 +758,7 @@
 
 /* INITIAL_DBUFS - how many dbufs to preallocate
  */
-#define INITIAL_DBUFS 4000 /* preallocate 16 megs of dbufs */
+#define INITIAL_DBUFS 300 /* preallocate 16 megs of dbufs */
 
 /* MAXBUFFERS - increase socket buffers
  *
